### DISCUSSION TITLE
Is possible to add a section in the prompt that calls a style from the style tab?

### DISCUSSION THREAD

**Discussion by PeterJared** [lllyasviel/Fooocus#1934] [Q&A] [answered]
I know there is a folder, sdxl_styles, that contains the prompts used to call the styles, but it includes structures that I don't know how to include in the prompt/ I guess simply copy and paste is not the solution. Any examples how to use it in the prompt to call those styles?

**Comment by mashb1t** [comment, marked as answer]
@PeterJared please navigate to the Advanced > Style tab to find all styles. You can find a preview of each style when hovering over them. Styles can be added to the prompt by just checking them.

What might be also interesting for you are wildcards:
A video guide about wildcards can be found in https://github.com/lllyasviel/Fooocus/discussions/1667 or find the quick version here https://github.com/lllyasviel/Fooocus/discussions/494#discussioncomment-7119075

**Reply by PeterJared** [reply]
What I want to do, if  possible, is a wildcard file that calls all or a selection of the styles without using the style tab. But even that file I mention has the prompts that trigger those styles, I don't understand what I should include from the files in sdxl_styles folder to add the wildcards that trigger the styles as  the structure of the files look no evident to be reduced to wildcards.

**Reply by mashb1t** [reply]
this is currently not possible without individual code modifications, as wildcards only apply positive prompts, not negative ones.
You might be better off using the Fooocus API to do so or by implementing a butto to randomly select 2-3 styles.
